services/management/checks.go

```diff
@@ -73,3 +65,69 @@ func (s *ChecksAPIService) GetSecurityCheckResults() (*managementpb.GetSecurityC
 
 	return &managementpb.GetSecurityCheckResultsResponse{Results: checkResults}, nil
 }
+
+// StartSecurityChecks executes Security Thread Tool checks and returns when all checks are executed.
+func (s *ChecksAPIService) StartSecurityChecks(ctx context.Context) (*managementpb.StartSecurityChecksResponse, error) {
+	err := s.checksService.StartChecks(ctx)
+	if err != nil {
+		if err == services.ErrSTTDisabled {
+			return nil, status.Errorf(codes.FailedPrecondition, "%v.", err)
+		}
+
+		return nil, errors.Wrap(err, "failed to start security checks")
+	}
+
+	return &managementpb.StartSecurityChecksResponse{}, nil
+}
+
+// ListSecurityChecks returns a list of available Security Thread Tool checks and their statuses.
+func (s *ChecksAPIService) ListSecurityChecks() (*managementpb.ListSecurityChecksResponse, error) {
+	disChecks, err := s.checksService.GetDisabledChecks()
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to get disabled checks list")
+	}
+
+	m := make(map[string]struct{}, len(disChecks))
+	for _, c := range disChecks {
+		m[c] = struct{}{}
+	}
+
+	checks := s.checksService.GetAllChecks()
+	res := make([]*managementpb.SecurityCheck, 0, len(checks))
+	for _, c := range checks {
+		_, disabled := m[c.Name]
+		res = append(res, &managementpb.SecurityCheck{Name: c.Name, Disabled: disabled})
+	}
+
+	return &managementpb.ListSecurityChecksResponse{Checks: res}, nil
+}
+
+// ChangeSecurityChecks enables/disables Security Thread Tool checks by names.
+func (s *ChecksAPIService) ChangeSecurityChecks(req *managementpb.ChangeSecurityChecksRequest) (*managementpb.ChangeSecurityChecksResponse, error) {
+	var enableChecks, disableChecks []string
+	for _, check := range req.Params {
+		if check.Enable && check.Disable {
+			return nil, status.Errorf(codes.InvalidArgument, "Check %s has enable and disable parameters set to the true.", check.Name)
+		}
+
+		if check.Enable {
+			enableChecks = append(enableChecks, check.Name)
```

Comment:
It is not really important but at this point we can just continue or replace the if by a switch.